packages/block-editor/src/layouts/flex.js

```diff
@@ -113,9 +119,11 @@ export default {
 		const flexWrap = flexWrapOptions.includes( layout.flexWrap )
 			? layout.flexWrap
 			: 'wrap';
+		const verticalAlignment =
+			verticalAlignmentMap[ layout.verticalAlignment ];
```

Comment:
You need to add a fallback to previous `center` value.

Reply:
Also the respective default for vertical orientation was the browser default so there was no visual issue, but in this case the `center` is not the default and we need to make the button active probably.

Reply:
> Also the respective default for vertical orientation was the browser default so there was no visual issue, but in this 
case the center is not the default and we need to make the button active probably.

I didn't really understand this part, could you please explain me in detail?

Reply:
It seems we just need [this change](https://github.com/WordPress/gutenberg/pull/40013/files#r842765973). Disregard my mention to browser defaults, because code might have changed here 😄 .

The point was that the controls should match the default/fallback value you added.